### DISCUSSION TITLE
Using "Microsoft Search" is not showing the RefinableString managed properties inside the "PnP search filter" web part

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#3427] [Q&A] [answered]
**I have created 2 SharePoint site columns of type managed metadata named "CustomDocumentType" & "CustomClient", as a result i got those 2 managed properties :-

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/2b6aa857-bf86-436a-bb66-6b8d1cb6eedb)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/41ca8c8b-5f08-4578-a2ef-18029bc730c3)

and those 2 crawl properties:-

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/f66975f5-0c88-403e-b194-e7670d926853)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/b0153e8a-56c4-4766-ae6d-33e64edbd2f6)

then i defined the 2 crawl properties inside the RefinaleString09 and RefinableString10, as follow:-

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/c4c75bf5-13d4-4cde-80b0-4a7cc9cd43a7)

then i added "PnP Search Filter" web part which is linked to a "PnP Search Result" web part which uses "Microsoft Search", as follow:-

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/3a9b59e5-b99b-423f-bb05-cb751af25bfa)

but the RefinaleString09 and RefinableString10 did not show inside the Filter fields inside the "PnP Search Filter" web part? any advice?

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/fcab5cf4-226c-4c9d-a5d8-fc3fdcd3b063)**

now i want to filter the documents using those 2 site columns? not sure how i need to configure the web parts?

**Comment by kasperbolarsen** [comment]
As I understand it RefinableString/Date and so on only exists in the SharePoint Search Schema, not in the Microsoft Search schema, as per https://github.com/microsoft-search/pnp-modern-search/issues/3138

**Comment by mvcsharepointdev** [comment, marked as answer]
@kasperbolarsen after 24 hours ,, I manually enter the Refinable09 & Refinable10 inside the Filter Fields dropdown and it worked well.. not sure if this is a hack or things suppose to work in this way:-

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/7688e272-54da-4269-b411-9e2b59eb6e2b)


here is the result:-

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/313859b3-437c-4e75-8540-72549af4341c)

**Comment by kasperbolarsen** [comment]
Ahh, makes sense since the data you are using is in SharePoint. Besides "kicking the tires" on Microsoft Search, why are you using Microsoft Search as the datasource rather than SharePoint Search ?

**Reply by mvcsharepointdev** [reply]
@kasperbolarsen because Microsoft search is more future proof compared to SharePoint search.. i think Microsoft Search is the same as Graph API is that correct?
